canvas_account_admin_tools/templatetags/json_filters.py

```diff
@@ -0,0 +1,12 @@
+import json
+
+from django.utils.safestring import mark_safe
+from django.template import Library
+
+
+register = Library()
+
+
+@register.filter(is_safe=True)
+def jsonify(object):
+    return mark_safe(json.dumps(object))
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Use of mark_safe() may expose cross-site scripting vulnerabilities and should be reviewed. https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html#b308-mark-safe